_includes/community_partners.html

```diff
@@ -1,25 +1,23 @@
 <section id="community-partners" class="wrapper speakers-section">
   <h2 class="special-bg" aria-label="Community Partners">Community Partners</h2>
  
- <div class="speaker-container">
+  <div class="row">
+    <div class="col-md-6">
```

Comment:
unfortunately the bootstrap styles are not added in current project, so this class might not work.